lib/workers/repository/process/extract-update.ts

```diff
@@ -129,7 +129,10 @@ export async function lookup(
     config,
     packageFiles
   );
-  logger.debug({ config: packageFiles }, 'packageFiles with updates');
+  logger.debug(
+    { baseBranch: config.baseBranch, config: packageFiles },
+    'packageFiles with updates'
+  );
```

Comment:
here `baseBranch` was added manually regardless of `baseBranches`

Reply:
I think that's ok. We log it once per base branch, right?

Reply:
yep